packages/clerk-js/src/core/clerk.ts

```diff
@@ -156,6 +163,7 @@ export class Clerk implements ClerkInterface {
   protected internal_last_error: ClerkAPIError | null = null;
 
   #publishableKey: string = '';
+  #cache = new ClerkMap();
```

Comment:
I do like the idea of this 👍 Seems like it'll be a really useful tool given the split between UI rendered by Clerk and the host.

That said, I'm not super excited about adding more to the public API of our core Clerk class. Given that this cache is pretty squarely a UI concern / helper, I wonder if it's a good time to introduce a `Clerk.ui` module to encapsulate these concerns. What do you think?

Reply:
Not sure how the Clerk.ui module will help us here.

Maybe we should simply introduce `Clerk.__internal_setRequestCache()` or `Clerk.__internal_requestCache` (setter/getter)

in order to avoid passing the cache as an option when calling `Clerk.load`.

To answer to your question, i do see the point of having another way of grouping these APIs, but I would not delay the PR for this.

Reply:
It helps by giving us a way to not continue to grow the top-level API surface area of our core Clerk class. That's fine if we don't want to delay 👍